app/templates/admin/manageDepartments.html

```diff
@@ -57,7 +58,7 @@ <h1>Manage Departments</h1>
               <tbody>
                 {% for department in activeDepartments %}
                 <tr>
-                  <td scope="row">{{department.DEPT_NAME}} <small>({{department.ORG}}, {{department.ACCOUNT}})</small></td>
+                  <td scope="row"><a href="#" class="supervisorDeptModal" id="{{department.departmentID}}">{{department.DEPT_NAME}}</a><small>({{department.ORG}}, {{department.ACCOUNT}})</small></td>
```

Comment:
This is a little more of a personal preference but currently, I dislike how when you click a link it changes the URL at the top of the page to be `/admin/manageDepartments#` instead of leaving it the same. Consider changing the `href=` to `javascript:{}` which doesn't do anything to change the url.